cmd/root.go

```diff
@@ -195,16 +184,15 @@ func outputJSON(results []pkg.Result) {
 	}
 
 	for _, r := range results {
-		var details []string
+		tmp_result := lib.CheckResult{
```

Comment:
tmpResult for var names in go (camelCase)

Reply:
Done.